wger/nutrition/migrations/0003_auto_20190618_1549.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
@Manorlds-Eaglespark This file looks misplaced and not related to your story scope.